src/XR.js

```diff
@@ -356,7 +326,7 @@ class XRWebGLLayer {
     this.alpha = alpha;
     this.multiview = multiview;
```

Comment:
this can be removed per https://github.com/immersive-web/webxr/commit/bb483c993df49e94c3660e6a7b046885d1570779

Reply:
Thanks!